Python/pySeaboard.py

```diff
@@ -53,19 +53,19 @@ def __call__(self, event, data=None):
     byte0 = midiData[0]
 
     if byte0 >= MIDI_OFF and byte0 < MIDI_ON:
-      print "Note Off"
+      print ("Note Off")
```

Comment:
Could we drop the space before the `(` to match the style used elsewhere in this file?